applications/GeoMechanicsApplication/custom_elements/README.md

```diff
@@ -0,0 +1,130 @@
+
+$$ \rho^w = 9.998396 \cdot 10^2 + 6.764771  \cdot 10^{-2} \cdot T - 8.993699  \cdot 10^{-3} \cdot T^2 + 9.143518 \cdot 10^{-5} \cdot T^3 - 8.907391 \cdot 10^{-7} \cdot T^4 + 5.291959  \cdot 10^{-9} \cdot T^5 - 1.359813  \cdot 10^{-11} \cdot T^6 $$
+
+Note: the dependency of water density and viscosity to temperature is user defined and the user can turn this feature on or off in the JSON file.
+
+## Finite Element Formulation
+
+Kratos solves the equations based on an incremental method. In the frame of Generelized Newmark method (GN11), the fully implicit incremental temperature formulation read as,
+
+$$ \left(\frac{1}{\theta \Delta t} \boldsymbol{S} + \boldsymbol{A} + \boldsymbol{H} + \boldsymbol{W}^l  \right) \boldsymbol{\Delta T} = \left( \frac{1}{\theta} - 1 \right) \boldsymbol{S} \frac{dT^n}{dt} - \left(\boldsymbol{A} + \boldsymbol{H} + \boldsymbol{W}^l \right) \boldsymbol{T}^{n} + \left( \boldsymbol{V} + \boldsymbol{W}^r \right) $$
+
+with $\theta = 1$ this formulation leads to backward Euler formulation. It reads,
+
+$$ \left(\frac{1}{\Delta t} \boldsymbol{S} + \boldsymbol{A} + \boldsymbol{H} + \boldsymbol{W}^l  \right) \boldsymbol{\Delta T} = - \left(\boldsymbol{A} + \boldsymbol{H} + \boldsymbol{W}^l \right) \boldsymbol{T}^{n} + \left( \boldsymbol{V} + \boldsymbol{W}^r \right) $$
+
+Note: the user can choose either Newmark or backward Euler from the JSON file. 
+
+***Compressibility matrix***
+
+$$ \boldsymbol{S} = \int_{\Omega^e} \left( n S \rho^w c^w + \left(1-n\right) \rho^s c^s \right)^{n+1} \boldsymbol{N}^T  \boldsymbol{N} d \Omega $$
+
+***Convectivity matrix***
+
+$$ \boldsymbol{A} = \int_{\Omega^e} \left(\rho^w c^w\right)^{n+1}  \boldsymbol{N}^T \boldsymbol{q}^{T,n+1} \boldsymbol{\nabla N}   d \Omega $$
+
+***Conductivity matrix***
+
+$$ \boldsymbol{H} = \int_{\Omega^e} \boldsymbol{\nabla N}^T \boldsymbol{D}^{n+1} \boldsymbol{\nabla N} d \Omega $$
+
+***Neumann condition (conductive boundary)***
+
+$$ \boldsymbol{V} = \int_{\Gamma_2^{ep}}  f^{n+1} \boldsymbol{N}^T  d \Gamma $$
+
+***Robin condition (convective boundary)***
+
+$$ \boldsymbol{W^r} = \int_{\Gamma_3^{ep}}  g^{n+1} \boldsymbol{N}^T  d \Gamma $$
+
+$$ \boldsymbol{W^l} = \int_{\Gamma_3^{ep}}  \left( \rho^w c^w q_n \right)^{n+1} \boldsymbol{N}^T \boldsymbol{I} d \Gamma $$
+
+where
+
+- $\Delta t$ = time step $\mathrm{\left[s \right]}$
+- $\Delta T$ = temperature increment $T^{n+1} - T^n$ $\mathrm{\left[^\circ C \right]}$
+- $\boldsymbol N$ = shape function array $\mathrm{\left[ - \right]}$
+- $\theta$ = a coefficient in Newmark time integration $\mathrm{\left[ - \right]}$
+- $\Omega$ = domain region
+- $\Gamma$ = boundary region
+
+The supercripts $^l$ and $^r$ for Robin boundary condition indicate the left hands side (matrix) and righ hand side (vector), respectively. The supercripts $^e$ and $^{ep}$ for $\Omega$ and $\Gamma$ indicate values in the element volume and perpendicular to element boundaries, respectively.  
```

Comment:
Sorry Mohammed, found another typo.
supercripts -> superscripts   2 times on this line.

Almost there...